src/components/layout/CyVerseAppBar.js

```diff
@@ -721,6 +741,28 @@ function CyverseAppBar(props) {
                 </Drawer>
             </Hidden>
             <CyVerseAnnouncer />
+            <Popover
+                open={Boolean(anchorEl)}
+                anchorEl={anchorEl}
+                onClose={onUserMenuClose}
+                anchorOrigin={{
+                    vertical: "bottom",
+                    horizontal: "center",
+                }}
+                transformOrigin={{
+                    vertical: "top",
+                    horizontal: "left",
+                }}
+            >
+                <UserMenu
+                    baseId={build(ids.APP_BAR_BASE, ids.ACCOUNT_MI)}
+                    profile={userProfile}
+                    onLogoutClick={onLogoutClick}
+                    onManageAccountClick={() =>
+                        window.open("https://user.cyverse.org", "_blank")
```

Comment:
The CyVerse policy and about URLs are in a constant file, maybe the user portal URL should also be in one?

Reply:
Yup..missed this one.